R/Connection.R

```diff
@@ -176,6 +176,23 @@ setMethod(
   }
 )
 
+#' @details odbcConnectionColumns is routed through the SQLColumns ODBC
```

Comment:
Needs `` `odbcConnectionColumns` `` etc

Reply:
Thanks - tried to address.  Let me know if I missed anything